content/glossary/build-harness.md

```diff
@@ -7,6 +7,6 @@ terms:
 tags:
 - best-practice
 ---
-The primary benefit of using a [`build-harness`]({{< relref "tools/build-harness.md" >}}) is it allows for the consolidation of business logic related to building software. This allows to keep things DRY. Using a centralized repo that can be versioned and shared across multiple projects reduces long-term technical debt associated with building and releasing software by reducing maintenance effort.
+The primary benefit of using a [`build-harness`]({{< relref "reference/tools/build-harness.md" >}}) is it allows for the consolidation of business logic related to building software. This allows to keep things DRY. Using a centralized repo that can be versioned and shared across multiple projects reduces long-term technical debt associated with building and releasing software by reducing maintenance effort.
```

Comment:
I think maybe we create a single page called `tools.md` (rather than reference/tools/build-harness.md") because we mostly won't have that much content.